scope/Cargo.toml

```diff
@@ -68,6 +68,10 @@ ignore.workspace = true
 tracing-indicatif.workspace = true
 indicatif.workspace = true
 strip-ansi-escapes.workspace = true
+octocrab = "0.38.0"
+jsonwebtoken = "9.3.0"
+secrecy = "0.8.0"
+url = "2.5.0"
```

Comment:
We've been putting the deps in the root cargo.toml, and using workspace = true to pull them in. This was done at a point in time where we had multiple workspaces, which was undone for publishing reasons, but those are no longer required.

Reply:
Done, thanks for pointing this out.